src/Hazelcast.Net/Clustering/ClusterMembers.cs

```diff
@@ -221,7 +221,7 @@ public async Task RemoveConnectionAsync(MemberConnection connection)
                 // if we are not disconnecting, we can return - we are done
                 if (!disconnecting)
                 {
-                    _logger.LogDebug($"Removed connection {connection.Id.ToShortString()} to member {connection.MemberId.ToShortString()}, remain connected.");
+                    _logger.LogDebug($"Removed connection {connection.Id.ToShortString()} to member {connection.MemberId.ToShortString()}, remain {(_connected?"":"dis")}connected.");
```

Comment:
I haven't finished the ILogger Refactoring but for further usages, we may get into habit of using `(ifDebug/ifWarning)?.` and not to use string interpolations. fyi. After release, I'll also include these to refactoring.

Reply:
Guess we should either use `IfDebug()?.` here, or template strings `_logger.LogDebug("Connection {ConnectionId}...", id);` - want me to change it here, or will you take care of it with your refactoring?

Reply:
I'll refactor them, np. I'm changing #540 to draft for now. After this is merged, I'll cover it.